checks/class-image-size-check.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Check if the image sizes are larger than necessary
+ *
+ * @package Theme Check
+ */
+
+/**
+ * Check if the image sizes are larger than necessary
+ */
+class Image_Size_Check implements themecheck {
+	/**
+	 * Error messages, warnings and info notices.
+	 *
+	 * @var array $error
+	 */
+	protected $error = array();
+
+	/**
+	 * Check that return true for good/okay/acceptable, false for bad/not-okay/unacceptable.
+	 *
+	 * @param array $php_files File paths and content for PHP files.
+	 * @param array $css_files File paths and content for CSS files.
+	 * @param array $other_files Folder names, file paths and content for other files.
+	 */
+	public function check( $php_files, $css_files, $other_files ) {
+
+		checkcount();
+
+		foreach ( $other_files as $other_key => $otherfile ) {
+			/*
+			* Check if the file is an image.
+			* Silence read error if the file is too small. @see https://www.php.net/manual/en/function.exif-imagetype.php#79283.
+			*/
+			if ( @wp_get_image_mime( $other_key ) !== false ) {
```

Comment:
Can we live with the silence here or do I need to rewrite it to do the file size check _before_ checking if it is actually an image?